pkg/windows/connectivity.go

```
@@ -121,6 +142,12 @@ func (c *sshConnectivity) init() error {
 		return fmt.Errorf("unable to connect to Windows VM %s: %w", c.ipAddress, err)
 	}
 	c.sshClient = sshClient
+
+	if err := c.hostKeyValidator.PersistSeenKey(ctx); err != nil {
+		c.sshClient.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `c.sshClient.Close` is not checked

(errcheck)